tests/testthat/_snaps/table_egt05_qtcat.md

```diff
@@ -3,76 +3,117 @@
-          Change from Baseline                                              
-            n                         134           134            132      
-            <=30 msec              63 (47%)     80 (59.7%)      81 (61.4%)  
-            >30 to <=60 msec      14 (10.4%)      8 (6%)        11 (8.3%)   
-            >60 msec              57 (42.5%)    46 (34.3%)      40 (30.3%)  
-        WEEK 4 DAY 29                                                       
-          Value at Visit                                                    
-            n                         134           134            132      
-            <=450 msec            117 (87.3%)   103 (76.9%)    114 (86.4%)  
-            >450 to <=480 msec     7 (5.2%)     14 (10.4%)       6 (4.5%)   
-            >480 to <= 500 msec     4 (3%)       7 (5.2%)        3 (2.3%)   
-            >500 msec              6 (4.5%)      10 (7.5%)       9 (6.8%)   
-          Change from Baseline                                              
-            n                         134           134            132      
-            <=30 msec              79 (59%)     80 (59.7%)      79 (59.8%)  
-            >30 to <=60 msec       11 (8.2%)     7 (5.2%)       10 (7.6%)   
-            >60 msec              44 (32.8%)    47 (35.1%)      43 (32.6%)  
-        WEEK 5 DAY 36                                                       
-          Value at Visit                                                    
-            n                         134           134            132      
-            <=450 msec            107 (79.9%)   117 (87.3%)    112 (84.8%)  
-            >450 to <=480 msec    16 (11.9%)     5 (3.7%)       13 (9.8%)   
-            >480 to <= 500 msec    5 (3.7%)      9 (6.7%)        3 (2.3%)   
-            >500 msec              6 (4.5%)      3 (2.2%)         4 (3%)    
-          Change from Baseline                                              
-            n                         134           134            132      
-            <=30 msec             72 (53.7%)    82 (61.2%)      73 (55.3%)  
-            >30 to <=60 msec       10 (7.5%)     11 (8.2%)      11 (8.3%)   
-            >60 msec              52 (38.8%)    41 (30.6%)      48 (36.4%)  
+      Parameter                                                                           
+        Analysis Visit              Placebo     Xanomeline High Dose   Xanomeline Low Dose
+          Category                  (N=86)             (N=84)                (N=84)       
+      ————————————————————————————————————————————————————————————————————————————————————
+      QT Duration (msec)                                                                  
+        Week 2                                                                            
+          Value at Visit                                                                  
+            n                         272               196                    200        
```

Comment:
check for n

Reply:
hi @edelarua , can you please check this little n again, if additional filtering step is needed here, n is equal to the sum of below, that is correct. but I was under the impression that n should be at most N at all visits. @khatril please correct me if I was wrong. thanks!

Reply:
In that case, we need to do one of the following:
1. Also filter or split by timepoint (so there is one record per patient) since `n` counts the number of records, not patients.
2. Use a different or custom function to calculate these rows instead of `analyze_vars()`.
Let me know what you think!

Reply:
hi @edelarua , I dont think it is necessary to use a custom function to analyze these rows. 

that is why I think a filtering step is needed.

Reply:
see https://github.com/pharmaverse/admiral/issues/2561

and discussion in https://github.com/pharmaverse/admiral/discussions/2559, this output will need to be refreshed once the new data is out